scalafmt-tests/src/test/resources/test/ContinuationIndent.stat

```diff
@@ -37,3 +37,15 @@ class ExtendTest
     with D
     with E =>
 }
+<<< #1334 1
```

Comment:
It might be better to move these tests to `test/resources/default/indent.stat` or something because this behavior is nothing to do with `continuationIndent`?

Reply:
no problem. perhaps i misunderstand the meaning of continuation; i thought that since we are indenting because of the `.something` which continues from previous line, it should go in this file.